lib/pub_sub/testing/rails_event_store.rb

```diff
@@ -0,0 +1,9 @@
+module PubSub
+  module Testing
+    module RailsEventStore
+      def event_store
+        Rails.configuration.event_store
+      end
```

Comment:
I don't know what you think about new Ruby syntax, but it's tempting to use the endless method definition here.
```suggestion
      def event_store = Rails.configuration.event_store
```